### PR TITLE
containers minimization

### PR DESCRIPTION
**Pull request by sandrobonazzola**
- avoid to install docs
- cleanup dnf cache
- avoid generating unnecessary intermediate layers

### CODE REVIEW

**Review by mwperina — APPROVED** [review]
LGTM

File: tests/containers/build-base [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 # CRB is required for meson
-RUN dnf config-manager -y --set-enabled crb
 # EPEL is required for python3-flake8 and codespell
-RUN dnf install -y epel-release
 
-RUN dnf install \
+RUN dnf install -y dnf-plugin-config-manager && \
+    dnf config-manager -y --set-enabled crb && \
+    dnf install -y epel-release && \
+    dnf install --setopt=tsflags=nodocs \
```
Comment (engelmi) [inline tests/containers/build-base]: Could this be just `dnf install --nodocs`?

**Review by engelmi — APPROVED** [review]
Small question, otherwise LGTM

File: tests/containers/build-base (hunk 1 above)
Comment (rhatdan) [reply tests/containers/build-base]: You might want to add a dnf -y update \ to this prior to all of the others.

Comment (sandrobonazzola) [reply tests/containers/build-base]: yes, `--nodocs` should be an alias to that, updating.
Also adding the update command as suggested.

Comment (sandrobonazzola) [reply tests/containers/build-base]: added a second commit addressing above comments

**Review by Yarboa — APPROVED** [review]

**Pull request merged**